### ISSUE TITLE
Can't see OpenLayers option window in QGIS 3.0

### ISSUE THREAD

**Issue by 9ls1** [tomchadwin/qgis2web#623]
Thanks for a great plug-in.
I've installed qgis2web in QGIS 3.0 and added the layers I want.
However, no OpenLayres nor Leaflet bakground map is appearing.
I can't see the "map option window" like when using qgis2web in QGIS 2.16.3.
**QGIS 2.16.3**:
![2018-03-08 12_07_46-export2webmap_2163](https://user-images.githubusercontent.com/10632163/37148167-5ba3e174-22c1-11e8-90d9-ad92aea24f0b.png)
**QGIS 3.0**:
![2018-03-08 12_07_46-export2webmap_30](https://user-images.githubusercontent.com/10632163/37148168-5d933a66-22c1-11e8-9106-9b1de6695e16.png)
Something I've missed?

**Comment by boesiii** [comment]
The background map options have been removed.  It now reads the background map you are using in your project.  You can easily add the OpenStreetMap by using the QGIS Browser panel.  Expand the XYZ Tiles and you will see the OpenStreetMap layer.  Drag that into your map and fire up qgis2web and you should see the OSM background.

**Comment by HerbFargus** [comment]
Also detailed in wiki:

[https://github.com/tomchadwin/qgis2web/wiki/basemaps](https://github.com/tomchadwin/qgis2web/wiki/basemaps)

**Comment by 9ls1** [comment]
Yes, I missed something! Rather embarrassing. I came across the plugin from here : http://www.qgistutorials.com/de/docs/web_mapping_with_qgis2web.html, tried it in QGIS 2.16.3. It worked and just tried the same in in 3.0. I just went (too) quickly through the wiki - but I, obviously, didn't scroll down far enough on the ‘base maps’-page. Sorry. Anyway, thanks a lot for your quick reply. Now the base map appear. :-)
